src/sql/workbench/contrib/editData/browser/media/editData.css

```diff
@@ -3,10 +3,11 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-.editdatagridpanel * {
+.slickgridContainer {
```

Comment:
This is too generic since it'll apply to all slick grid containers in the app. Please scope it down to just the edit data component